group_project_v2/stage/base.py

```diff
@@ -7,20 +8,29 @@
 from xblock.fragment import Fragment
 from xblockutils.studio_editable import XBlockWithPreviewMixin
 from group_project_v2.api_error import ApiError
-from group_project_v2.mixins import CommonMixinCollection, XBlockWithUrlNameDisplayMixin, AdminAccessControlXBlockMixin
+from group_project_v2.mixins import (
+    CommonMixinCollection, XBlockWithUrlNameDisplayMixin, AdminAccessControlXBlockMixin,
+    DashboardXBlockMixin, DashboardRootXBlockMixin
+)
 from group_project_v2.notifications import StageNotificationsMixin
 from group_project_v2.stage_components import (
     GroupProjectResourceXBlock, GroupProjectVideoResourceXBlock, ProjectTeamXBlock
 )
 from group_project_v2.utils import (
     gettext as _, HtmlXBlockShim, format_date, Constants, loader,
-    outsider_disallowed_protected_view, add_resource, MUST_BE_OVERRIDDEN, get_link_to_block
-)
+    outsider_disallowed_protected_view, add_resource, MUST_BE_OVERRIDDEN, get_link_to_block,
+    get_block_content_id)
```

Comment:
@bradenmacdonald I'm trying very hard to pt them on next line, but PyCharm "automatic import" have a different opinion on that. Thanks for catching this